### PR TITLE
API-6710: Update support to help guide veterans

### PR DESCRIPTION
**Pull request by anthony-kopczyk**
### Description

Jira Ticket: https://vajira.max.gov/browse/API-6710

This set of changes is intended to help guide them to va.gov or their local VSO to find data, since the portal contact us page is intended for developers and API questions

### Process

- [x] Tests added or updated for change
- [N/A] Docs updated
  - [N/A] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated
  - [N/A] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated
- [x] Accessibility concerns have been considered and addressed
  
  - [x] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues

### Requested feedback

Specifically double checking to make sure this is consistent with the content in the jira ticket.

### CODE REVIEW

**Comment by gileswells** [comment]
@anthony-kopczyk does this pass visual regressions? I know the npm audit currently fails but with on page changes I'm expecting those to fail.

**Comment by anthony-kopczyk** [comment]
> @anthony-kopczyk does this pass visual regressions? I know the npm audit currently fails but with on page changes I'm expecting those to fail.

Oh! I didn't think of running them/forgot about them. They should fail with these changes. I'll get this updated with updated files. Thanks!

File: src/containers/support/ContactUs.tsx [hunk 1]
```diff
@@ -32,16 +33,14 @@ const ContactUs = (): JSX.Element => {
...
-          </p>
-          <p>
-            <strong>Note:</strong> If you are a Veteran seeking help with claims, health records, or
-            other information, visit <a href="https://www.va.gov/">VA.gov</a> or contact your local
-            VSO for assistance.
-          </p>
+          <p>If you have questions about APIs, development, or related topics, use this form to send us a message.</p>
+          <AlertBox
+            status="info"
+            key="contactus"
```
Comment (gileswells) [inline src/containers/support/ContactUs.tsx:39]: Is there a reason you're using a key here? From my testing you don't need it and if that's the case you can move the entirety of the `<AlertBox />` component into the `ContactUsAlertComment.tsx` file.

**Review by gileswells — CHANGES_REQUESTED** [review]
Unless there's a reason I'm missing for using the key I'd much prefer to see it bundled up a little bit more since it's all the same markup wrapping the actual text.

File: src/containers/support/ContactUs.tsx (hunk 1 above)
Comment (anthony-kopczyk) [reply src/containers/support/ContactUs.tsx:39]: That comes from copy/pasta when using this component. I pulled it from the APIDocs or something like that (haven't done the alert box before, and now it's fairly apparent to me I didn't spend enough time making sure I knew the component).

I'll look at that component and try to tidy it up. Being able to move the entire list box into that other file will make it much cleaner too.

Thanks!

**Review by gileswells — APPROVED** [review]
LGTM

**Pull request merged**